hazelcast/include/hazelcast/client/serialization/pimpl/compact/compact_impl.h

```diff
@@ -596,18 +652,72 @@ T inline compact_stream_serializer::read(object_data_input& in)
         };
         BOOST_THROW_EXCEPTION(exception);
     }
+
     compact_reader reader = create_compact_reader(*this, in, schema);
     return hz_serializer<T>::read(reader);
 }
 
+generic_record::generic_record compact_stream_serializer::read_generic_record(object_data_input& in)
```

Comment:
read_generic_record is made inline in class declaration while write_generic_record is made inline in method definition. Is there a specific reason for this? It makes the reader to read the code as one is inline, and the other is not.

Reply:
Fixed.